src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const UniqueValues = new Set();
+  const Values = Object.values(robot);
```

Comment:
it should not be capitalized (capitalized variables refer to classes, here's just objects
```suggestion
  const uniqueValues = new Set();
  const values = Object.values(robot);
```